core/kazoo_services/src/services/kz_service_phone_numbers.erl

```diff
@@ -57,6 +58,18 @@ reconcile(Services, PNs) ->
     S2 = kz_services:reset_category(?NUMBER_SERVICES, S1),
     update_numbers(S2, PNs).
 
+-spec reset(kz_services:services()) -> kz_services:services().
+reset(Services) ->
+    Categories = [?PHONE_NUMBERS
+                 ,?NUMBER_SERVICES
```

Comment:
What about the other two categories?
I'm thinking we can put the map as a macro and use it to get the list of categories to reset.
Also, the function just above shares the same body as this one